substrate/frame/examples/kitchensink/src/lib.rs

```diff
@@ -226,6 +226,37 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// A call that is specially authorized.
+		/// Authorized call can be dispatched by anybody without requiring any signature or fee.
+		#[pallet::call_index(1)]
+		#[pallet::authorize(|
```

Comment:
These authorized calls will not work in batch calls for example?

Reply:
For now it would not work batch call, because `pallet-utility` doesn't give an authorization function for the call batch.

`pallet-utility` could provide an authorization function for the call `batch` by checking that all inner call are authorized. But I worry it makes the validity check unbonded.

In general, `authorize` function are more for single transaction like `claim` or `provide_election_solution` or `tasks`.
When the behavior is about allowing an origin to do some call, I think it should use its own origin.
Right we mostly only use `Signed` origin, but a pallet could have a new transaction extension that mutate the origin to a `Person` origin, then this `Person` origin can use batch without issue.